.travis.yml

```
@@ -84,23 +89,26 @@ install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
     pip install --upgrade pip wheel;
     pip install --upgrade numpy;
-    pip install .;
-    pip uninstall opencv-contrib-python -y;
+    pip install .[asyncio];
+    pip uninstall opencv-python -y;
     pip install six;
     pip install codecov;
+    pip install --upgrade flake8;
     pip install --upgrade pytest;
+    pip install --upgrade pytest-asyncio;
     pip install --upgrade pytest-cov;
     pip install --upgrade youtube-dl;

```

Now that pip is getting a real dependency resolver (pypa/pip#988), you should hand it the requirements in a single command.
```suggestion
    pip install .[asyncio] codecov flake8 numpy opencv-python pytest pytest-asyncio pytest-cov six youtube-dl;
```